setup.py

```diff
@@ -5,11 +5,12 @@
 
 if __name__ == "__main__":
     if not isfile(config_path):
-        print("Missing config files in {}. First copy files from configs/".format(root_path))
-        print("For example, for setting up backups on the support machine:")
-        print("mkdir -p {0} && cp configs/support.montagu/* {0}".format(root_path))
+        print("Expecting a file at " + config_path)
+        print("Missing config file. First copy file from configs/".format(root_path))
+        print("For example, for setting up backups on the annex machine:")
+        print("cp configs/annex.json config.json")
         exit(-1)
 
     print("Obtaining secrets from the vault. If you are not authenticated with the vault, this will fail.")
     settings = load_settings()
-    save_secrets(settings.encrypted)
+    save_secrets(any(t for t in settings.targets if t.encrypted))
```

Comment:
Is it worth checking at this point that there are no duplicate entries in target bucket (and possibly source paths?)

Reply:
I've added a check for IDs in the `Settings` constructor. Paths is more complicated, as each target has its own way of defining that, and some of them can't be resolved until the docker volume has been created etc.